src/media/js/helpers_local.js

```diff
@@ -136,8 +136,11 @@ define('helpers_local',
         format: format.format,
         getReviewId: getReviewId,
         has_installed: has_installed,
+        isDesktop: function() {
```

Comment:
And then we wouldn't need this in helpers...although it could go in utils or get shoved into capabilities.

Eventually, we could probably have a source of truth for viewport breakpoints and use that to generate a JS file and Stylus lib file.